source/modules/MEI.shared.xml

```diff
@@ -5524,6 +5524,17 @@
       </rng:zeroOrMore>
     </content>
     <attList>
+      <attDef ident="func" usage="opt">
+        <desc xml:lang="en">Indicates whether the element acts as a measure or a system within the flow.</desc>
```

Comment:
```suggestion
        <desc xml:lang="en">Indicates whether the element acts like a measure or a system within the flow.</desc>
```

Reply:
Not sure about this one. Maybe an native English speaker can decide.

Reply:
@ahankinson ?

Reply:
@ahankinson ?

Reply:
I think that both are valid alternatives; while `act as` emphasises a role, `act like` emphasises a behaviour. But what about adding a bit more verbosity, e.g.:
```suggestion
        <desc xml:lang="en">Specifies how the element is treated in terms of page flow: as inline element like a measure, or as block element like a system.</desc>
```